src/app.js

```diff
@@ -1 +1,36 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const [source, destination] = process.argv.slice(2);
+
+function copyFile(copyFrom, copyTo) {
+  try {
+    fs.existsSync(copyFrom);
```

Comment:
The `fs.existsSync(copyFrom);` line is not being used correctly. This function returns a boolean indicating whether the file exists, but here it is not being checked. You should use it in a conditional statement to handle the case where the file does not exist.